File: C#/Code/SPTemplate.UI/VisualWebParts/SampleVisualWebPart/SampleVisualWebPart.cs
using System;
using System.ComponentModel;
using System.Web;
using System.Web.UI;
using System.Web.UI.WebControls;
using System.Web.UI.WebControls.WebParts;
using Microsoft.SharePoint;
using Microsoft.SharePoint.WebControls;
using $customUsing$.UI.WebControls;

namespace $safeprojectname$.WebParts
{
    [ToolboxItemAttribute(false)]
    public class SampleVisualWebPart :   BaseWebPart
    {
        // Visual Studio might automatically update this path when you change the Visual Web Part project item.
        private const string _ascxPath = @"~/_CONTROLTEMPLATES/ProjectFolder/HVWP.ascx";

        protected override void CreateChildControls()
        {
            Control control = Page.LoadControl(_ascxPath);
            Controls.Add(control);
        }
    }
}

File: trunk/VB/Code/SPTemplate.UI/VisualWebParts/SampleVisualWebPart/SampleVisualWebPart.vb
﻿Imports System
Imports System.ComponentModel
Imports System.Web
Imports System.Web.UI
Imports System.Web.UI.WebControls
Imports System.Web.UI.WebControls.WebParts
Imports Microsoft.SharePoint
Imports Microsoft.SharePoint.WebControls

Namespace WebParts
    <ToolboxItemAttribute(False)> _
    Public Class SampleVisualWebPart
        Inherits WebPart

        ' Visual Studio might automatically update this path when you change the Visual Web Part project item.
        Private Const _ascxPath As String = "~/_CONTROLTEMPLATES/ProjectFolder/SampleVisualWebPart/HVWP.ascx"

        Protected Overrides Sub CreateChildControls()
            Dim control As Control = Page.LoadControl(_ascxPath)
            Controls.Add(control)
        End Sub

    End Class
End Namespace

